Tutorial_1a_Convolving_Images.ipynb

```diff
@@ -0,0 +1,693 @@
+{
```

Comment:
<p>Typo in constructing. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>